cross/python310/Makefile

```diff
@@ -133,24 +140,10 @@ python310_post_install: $(WORK_DIR)/python-cc.mk
 	cp -R $(HOSTPYTHON_LIB_NATIVE) $(PYTHON_LIB_CROSS)/../
 	@$(RUN) $(PYTHON_NATIVE) -m crossenv $(STAGING_INSTALL_PREFIX)/bin/python$(PKG_VERS_MAJOR_MINOR) $(WORK_DIR)/crossenv/
 	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) wget https://bootstrap.pypa.io/get-pip.py
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-python get-pip.py --no-setuptools --no-wheel
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) python get-pip.py --no-setuptools --no-wheel
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "setuptools==62.4.0"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "setuptools==62.4.0"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "setuptools-scm==6.4.2"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "setuptools-scm==6.4.2"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "wheel==0.37.1"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "wheel==0.37.1"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "cffi==1.15.0"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "cffi==1.15.0"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "cryptography==3.3.2"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "cryptography==3.3.2"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "poetry==1.1.13"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "poetry==1.1.13"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "Cython==0.29.28"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "Cython==0.29.28"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "flit==3.7.1"
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip install "flit==3.7.1"
+	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-python get-pip.py "pip==22.0.4" --no-setuptools --no-wheel --disable-pip-version-check
+	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip --disable-pip-version-check install "setuptools==62.4.0" "setuptools-scm==6.4.2" "wheel==0.37.1" "cffi==1.15.0" "cryptography==3.3.2" "poetry==1.1.13" "Cython==0.29.28" "flit==3.7.1"
+	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) python get-pip.py "pip==22.0.4" --no-setuptools --no-wheel --disable-pip-version-check
+	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) pip --disable-pip-version-check install "setuptools==62.4.0" "setuptools-scm==6.4.2" "wheel==0.37.1" "cffi==1.15.0" "cryptography==3.3.2" "poetry==1.1.13" "Cython==0.29.28" "flit==3.7.1"
```

Comment:
I like that you kept `--no-setuptools --no-wheel` that @Safihre used.  This gives a more granular management of versions as I did encountered issues in the past.